### DISCUSSION TITLE
Set drive for huggingface cache

### DISCUSSION THREAD

**Discussion by Bluebizzer** [nod-ai/SHARK#1207] [Q&A] [answered]
I see it says how to set the shark_tank cache, but I'm not sure how to set the cache dir for the huggingface cache. It's currently defaulting to the C:/ drive, which isn't optimal. 

Is there a similar flag for setting huggingface cache? 

Otherwise everything has been great, thanks for the work put into it. 

Thanks.

**Comment by powderluv** [comment, marked as answer]
Please use HF_HOME environment variable. 

Thanks for using SHARK

**Reply by Bluebizzer** [reply]
Perfect, thank you! :)
